Fabric.Terminology.API/Fabric.Terminology.API.csproj

```diff
@@ -9,6 +9,8 @@
     <RepositoryUrl>https://github.com/HealthCatalyst/Fabric.Terminology</RepositoryUrl>
     <PackageLicenseUrl>https://github.com/HealthCatalyst/Fabric.Terminology/blob/develop/LICENSE</PackageLicenseUrl>
     <AssemblyName>Fabric.Terminology.API</AssemblyName>
+    <CodeAnalysisRuleSet>Fabric.Terminology.API.ruleset</CodeAnalysisRuleSet>
+    <RunCodeAnalysis>false</RunCodeAnalysis>
```

Comment:
Are you sure you dont want code analysis?

Reply:
I do (eventually), but there is an issue with the DotNet Core 2 projects and the analyzers and this was the recommended solution.  You will find if you set this to true, the analyzer will run but the project compilation will fail.  I had done this for each project, tweaked any issues and then reset to false to allow the build to complete.